src/Symfony/Component/Form/Tests/Resources/TranslationFilesTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Tests\Resources;
+
+class TranslationFilesTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @dataProvider provideTranslationFiles
+     */
+    public function testTranslationFilesAreValid($filePath)
```

Comment:
technically, this test concerns a single file, so you should use a singular in the test name (yeah, this is nit-picking :smile: )